src/pages/TimetableManagement/page.tsx

```
@@ -38,6 +45,38 @@ export default function Page() {
+                    </TableHeaderText>
+                </TableHeader>
+            ))
+        ],
+        gap: '16px'
+    }
+    const row: RowInterace = {
+        count: DAYS_OF_WEEK.length + 1,
+        headNode: [
+            ...DAYS_OF_WEEK.map((rowName) => (
+                <TableHeader>
+                    <TableHeaderText color={colors.gray70}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)